autoware_launch/config/map/lanelet2_map_loader.param.yaml

```diff
@@ -0,0 +1,7 @@
+/**:
+  ros__parameters:
+    lanelet2_map_projector_type: UTM   # Options: MGRS, UTM
```

Comment:
Why is this projector type is changed from the default parameter in autoware.universe/map/map_loader/config ?